.buildkite/pipeline.yml

```diff
@@ -75,7 +75,7 @@ steps:
       buildkite-agent artifact download "*\\RELEASES" .
 
       echo "--- :node: Generating Release Manifest"
-      install_npm_packages
+      .buildkite/commands/install-node-dependencies.sh
```

Comment:
The `install_npm_packages` is [already a script that handles caching, defined in our `a8c-ci-toolkit` plugin](https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/blob/trunk/bin/install_npm_packages).

Would it make sense to add the handling of `patches/` in the `CACHE_KEY`… in the script from the plugin, so that other projects can benefit from it too? Or is the handling of `patches/*` and also what `install-node-dependencies.sh` does is too specific to `studio` to be worth fixing upstream too?

Reply:
It makes sense, but as four steps use custom `install-node-dependencies.sh` and two steps use a more generic command from upstream, I would prefer to fix it by using the consistent custom script now and bringing those changes to upstream separately if it benefits other projects that use that.

The number of changes is bigger than cache key handling e.g. one uses `npm ci` vs `npm install`.

Reply:
Added https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/issues/94